Phat-Taupe-Pests/BangazonAPI
# Ben payment types

***********

**Description**

Wrote the model and controller for Payment Types.
Tested GET, POST, PUT, and DELETE.

***********

**Status**

Ready for testing.

***********

**Steps to test**

1. Stash any local changes and switch to this branch with `git checkout -b BenPaymentTypes`
2. `git pull origin BenPaymentTypes`
3. `dotnet restore` and `dotnet run`
4. Test functionality using Postman or other. Route: https://localhost:5000/paymenttype
5. Payment type requires a customer foreign key. Create a customer by POSTing `{"firstName": "Ben", "lastName" : "Greaves"}`.
6. PaymentType has 3 required fields: string "Name", int "AccountNumber", and int CustomerID (FK tied to Customer).
7. Test POST, PUT, DELETE, GET. Note that the AccountNumber field is limited to 10 numbers.

***********

**Reference**

Ticket - https://github.com/Phat-Taupe-Pests/BangazonAPI/issues/3
	
***********
